content/chainguard/chainguard-images/getting-started/wordpress.md

```diff
@@ -0,0 +1,305 @@
+
+The main difference here is that we're calling the entrypoint script at **build time** instead of run time. This is done to ensure the image is self-contained and doesn't rely on volumes set up within the host machine in order to work. Any customizations should be included in the `wp-content` folder that will be copied to the image at build time. Although this increases final image size due to the inclusion of custom content at build time, it limits what can be changed or added to the image once it's built.
+
+This demo includes a theme ([cue](https://wordpress.org/themes/cue/), a simple blogging theme) and a plugin ([imsanity](https://wordpress.org/plugins/imsanity/), a popular plugin used to resize images) to demonstrate how to include custom content in the image.
+
+Navigate to the `03-distroless` directory to follow along. This is how the Dockerfile included in this directory looks like:
+
+```Dockerfile
+FROM cgr.dev/chainguard/wordpress:latest-dev as builder
+#trigger wp-config.php creation
+ENV WORDPRESS_DB_HOST=foo
+
+#copy wp-content folder
+COPY ./wp-content /usr/src/wordpress/wp-content
+
+#run entrypoint script
+RUN /usr/local/bin/docker-entrypoint.sh php-fpm --version
+
+FROM cgr.dev/chainguard/wordpress:latest
+
+COPY --from=builder --chown=php:php /var/www/html /var/www/html
+```
+
+Notice that we're copying the contents of the local `wp-content` folder to the `/usr/src/wordpress` folder in the container. This is the location of the WordPress source files. These will be copied to the document root by the entrypoint script that is executed right afterward. At the `builder` stage, we're also setting up a single environment variable to trigger the creation of the `wp-config.php` file that relies on a set of environment variables to configure database access.
+
+In the `docker-compose.yaml` file, we reference the custom Dockerfile:
+
+```yaml
+services:
+  app:
+    image: wordpress-local-distroless
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    environment:
+      WORDPRESS_DB_HOST: mariadb
+      WORDPRESS_DB_USER: $WORDPRESS_DB_USER
+      WORDPRESS_DB_PASSWORD: $WORDPRESS_DB_PASSWORD
+      WORDPRESS_DB_NAME: $WORDPRESS_DB_NAME
+    volumes:
+      - document-root:/var/www/html
+
+  nginx:
+    image: cgr.dev/chainguard/nginx
+    restart: unless-stopped
+    ports:
+      - 8000:8080
+    volumes:
+      - document-root:/var/www/html
+      - ./nginx.conf:/etc/nginx/nginx.conf
+
+  mariadb:
+    image: cgr.dev/chainguard/mariadb
+    restart: unless-stopped
+    environment:
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 1
+      MARIADB_USER: $WORDPRESS_DB_USER
+      MARIADB_PASSWORD: $WORDPRESS_DB_PASSWORD
+      MARIADB_DATABASE: $WORDPRESS_DB_NAME
+    ports:
+      - 3306:3306
+
+volumes:
+  document-root:
+
+```
+
+You can now build and run your environment with:
+
+```shell
+docker compose up --build
```

Comment:
It appears that both `docker compose` and `docker-compose` exist as commands... so far I've only noticed you using `docker-compose` in the commands. I'm not sure I understand the difference, but I'm flagging this since you might know, or so you can change it if it was unintentional.

Reply:
The file is `docker-compose.yaml`, but the command is `docker compose`

Reply:
Oh I see now there was a wrong `docker-compose` in the guide as command, that was probably a wrong autocomplete from my editor!